src/Controls/src/Core/Platform/Android/Extensions/FormattedStringExtensions.cs

```diff
@@ -64,6 +64,12 @@ public static SpannableString ToSpannableString(this Label label)
 
 			var builder = new StringBuilder();
 
+			var fontMetrics = context?.Resources?.DisplayMetrics != null ? new Paint()
+			{
+				TextSize = TypedValue.ApplyDimension(ComplexUnitType.Sp, (float)defaultFontSize, context.Resources.DisplayMetrics)
```

Comment:
This line looks like it would throw if `context.Resources` returned `null`.

Reply:
#I'm not sure why it would throw🤔
The only place that `fontMetrics` is  here https://github.com/dotnet/maui/blob/24d404991a5b7246dc05be856effea1ca0f3bf25/src/Controls/src/Core/Platform/Android/Extensions/FormattedStringExtensions.cs#L110 and there's a null check